src/index.html

```diff
@@ -5,10 +5,27 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="logotype">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <a href="#" class="is-active nav_item">Apple</a>
+          <a href="#" class="nav_item">Samsung</a>
```

Comment:
Let's make all these links more informative
```suggestion
          <a href="#samsung" class="nav_item">Samsung</a>
```